src/experimental/safe_html_transform/safe_html.py

```diff
@@ -0,0 +1,2630 @@
+import logging
```

Comment:
@tisto  So we have to remove all the condition from safe_html that include the logging functionalities. As of now we are assuming that our safe_html is registered, right ?